docs/arianee-js-certificate.md

```diff
@@ -517,5 +517,49 @@ Create an action Arianee Access Token link.
 const b=wallet.methods.createActionArianeeAccessTokenLink('http://www.authentificate-me.com',3703454);
 
 >
-http://www.authentificate-me.com/?arianeeAccessToken=eyJ0eXAiOiJKV1QiLCJhbGciOiJFVEgifQ%3D%3D.eyJpc3MiOiIweDQ5NjdmOTZBOTUyMDk2Q2IxODAyRDM3NWNCNzAzMTRmODcyOUE5NzciLCJzY29wZSI6ImFsbCIsImV4cCI6MTU5NTkyMjcyMTE5OCwiaWF0IjoxNTk1OTIyNDIxMTk4LCJjaGFpbiI6InRlc3RuZXQiLCJzdWIiOiJjZXJ0aWZpY2F0ZSIsInN1YklkIjozNzAzNDU0fQ%3D%3D.0x3789b08e6357bff40422889a5fde80b9ade2a610052a0453d76909a3e3b1502463d33a4929c646ba39fd6262187f0dd3a6934d09a38560822eda0d49e1550e8b1b
 ```
+
+### Is auth URL
+```
+ArianeeWallets.methods.isAuthURL(url)
+```
+
+Easily check if an URL is auth URL without having to check if it is a proof or an Arianee Access Token.
```

Comment:
Check if this URL is authenticated by either a proof or an Arianee access token.